app.js

```diff
@@ -22,4 +24,25 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(beersFromApi => {
+      console.log("Beers from database: ", beersFromApi)
+      res.render('beers', {beersList: beersFromApi});
+    })
+    .catch(error => console.log(error));
+});
+
+app.get('/random-beer', (req, res) => {
+  punkAPI
+    .getRandom()
+    .then(responseFromApi => {
+      console.log("Random Beer: ", responseFromApi)
+      res.render('random-beer', {randomBeer: responseFromApi});
```

Comment:
this works perfectly but as a suggestion, consider extracting the object from the array before passing it to the view to have a cleaner code ;)